src/collectd.conf.pod

```diff
@@ -8300,6 +8300,11 @@ Synopsis:
      CollectVCL         false
      CollectVSM         false
      CollectWorkers     false
+     CollectLCK         false
+     CollectMEMPOOL     false
```

Comment:
As above: Please use all-caps for acronyms / initialisms and find use CamelCase otherwise. I realize this doesn't make a difference since the code is matching cases-insensitive, but this being documentation I'd argue that readability is paramount.